src/config.ts

```diff
@@ -3006,6 +3012,48 @@ export const EmbedChainInfos: ChainInfoWithExplorer[] = [
 		features: ['stargate', 'ibc-transfer', 'no-legacy-stdTx', 'ibc-go'],
 		explorerUrlToTx: 'https://www.mintscan.io/fetchai/txs/{txHash}',
 	},
+    {
+		rpc: 'https://rpc.galaxychain.zone',
+		rest: 'https://rest.galaxychain.zone',
+		chainId: 'galaxy-1',
+		chainName: 'galaxy',
+		stakeCurrency: {
+			coinDenom: 'GLX',
+			coinMinimalDenom: 'uglx',
+			coinDecimals: 6,
+			coinGeckoId: '',
+			coinImageUrl: window.location.origin + '/public/assets/tokens/glx.svg',
+		},
+		bip44: {
+			coinType: 118,
+		},
+		bech32Config: Bech32Address.defaultBech32Config('galaxy'),
```

Comment:
Both here, and on the Chain Registry, bech32 prefix is 'galaxy', while registered to SLIP0173, it is 'glx'. Which is correct? if 'galaxy', please have the SLIP-173 registry corrected.
https://github.com/satoshilabs/slips/blob/master/slip-0173.md

Reply:
I have mixed in slip. correction made